app/views/offers/index.html.erb

```diff
@@ -1,2 +1,77 @@
-<h1>Offers#index</h1>
-<p>Find me in app/views/offers/index.html.erb</p>
+<!-- <div class="banner" id="map" style="background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url(https://api.mapbox.com/[...]?access_token=pk.eyJ1IjoiYW5ndXN0YXRjaGVsbCIsImEiOiJjanRlaWQwbzMwMTlsM3lydWMwNW41dG40In0.P4X8Ae7V6RXAGw65l6T-PA);">
+</div> -->
+
+
+
+<div class="banner" id="map" style="background-image: linear-gradient(rgba(0,0,0,0),rgba(0,0,0,0)), url(https://www.justinesmith.net/wordpress/wp-content/uploads/2017/05/World-Map-2013-2254x1360.jpg);">
+</div>
+
+<div class="container">
+
+<h1 style="margin-top: 20px;">Available Loans</h1>
+
+<div class="cards">
+  <% @offers.each do |offer| %>
+    <% link_to offer_path(offer.id) %>
+      <div class="card-trip">
+        <%= cl_image_tag offer.photo, height: 150, width: 200, crop: :fill %>
+        <div class="card-trip-infos">
+          <div>
+            <h2>Title here</h2>
+            <p>Short description here!</p>
+          </div>
```

Comment:
What will the title and desciption be? Can you make that dynamic before we merge it.